Automattic/wp-calypso
# Domains: Update transfer precheck per GDPR

- Remove e-mail/privacy precheck section
- Update the EPP code section to accept a code
- Validate code before adding to cart

Testing instruction:
- Calypso -> Add -> Use a Domain I Own
- Try a locked domain
- Make sure it shows proper message
- Unlock
- Recheck lock
- Get to EPP code input
- Input invalid EPP code
- No go
- Input valid EPP code
- Add product to cart
- Checkout
- Make sure the domain went straight to `pending_registry` status

Dependency: D13671-code
